README.md

```diff
@@ -3,11 +3,50 @@
 # :watch::horse: NightmareJS Plugin for Watchmen
 ## Allows Nightmare scripts to be executed by a Watchmen instance
 
+* [nightmare (@:octocat:)][nightmare]
+* [watchmen (@:octocat:)][watchmen]
+
 Feed a nightmare script to Watchmen to run synthetic monitoring.
 
-In Watchmen's `package.json` make sure to install [nightmareJS][nightmare].
-* [nightmare (@:octocat:)][nightmare]
-* [watchmen (@GH)](https://github.com/iloire/watchmen)
+In your Watchmen directory:
+`npm install nightmare --save`
+`npm install watchmen-ping-nightmare --save`
+
+
+### How To Use:
+- Create a nightmare script with a `.js` extension
+```javascript
+var Nightmare = require('nightmare');
+
+Nightmare
```

Comment:
or you can use alternate setup doc'd here https://github.com/segmentio/nightmare#usage
```javascript
var Nightmare = require('nightmare');
var nightmare = Nightmare();

nightmare.goto( /*etc...*/
```